lib/sles4sap_publiccloud.pm

```diff
@@ -129,11 +129,12 @@ sub run_cmd {
 
 sub get_promoted_hostname {
     my ($self) = @_;
+    my $master_resource_type = get_var('USE_SAP_HANA_SR_ANGI') ? "mst" : "msl";
```

Comment:
I don't see `mst_` resources in https://github.com/SUSE/qe-sap-deployment/pull/284. Are they coming later?